### ISSUE TITLE
Adding Morello support

### ISSUE THREAD

**Issue by rwatson** [CTSRD-CHERI/cheri-exercises#25]
On the whole, the exercises and missions in this document are portable across CHERI architectures. However, the cross-compilation tools/instructions and sample answers are specific to CHERI-RISC-V. Adding Morello variants, once the world is ready, would be quite useful.

**Comment by rwatson** [comment]
Closing as this is believed complete (although an ABI-related build issue remains; will create a separate ticket for that).
